pygn-mode.el

```diff
@@ -468,6 +468,28 @@ ignore the bundled library and use only the system `$PYTHONPATH'."
 
 ;;; Utility functions
 
+(defun pygn-mode-comment-region-contextually (beg end &optional arg)
+  "Wrap `comment-region-default' to remove `comment-continue' characters.
+
+This allows multi-line `comment-region' to work in `pygn-mode'
+without adding extra characters at beginning-of-line."
+  (if (and (pygn-mode--true-containing-node 'movetext beg)
+           (pygn-mode--true-containing-node 'movetext end))
+      (save-match-data
+        (comment-region-default beg end arg)
+        (save-restriction
+          (exchange-point-and-mark)
+          (narrow-to-region (min (point) (mark) beg) (max (point) (mark) end))
+          (save-excursion
+            (goto-char (point-min))
+            (while (re-search-forward (concat "^" (regexp-quote (or comment-continue "|"))) nil t)
```

Comment:
should `"^"` not be in `regexp-quote` function call?

Reply:
That would search for a literal `^`, but the code intends to look  for `comment-continue` at the starts of lines.

Reply:
Oh.